model_analyzer/config/config_value.py

```diff
@@ -85,23 +83,33 @@ def value(self):
         """
 
         return_result = self._value
-        if type(self._value) is dict:
-            return_result = {}
-            for key, value_ in self._value.items():
+        if self._output_mapper:
+            return_result = self._output_mapper(return_result)
+
+        if type(return_result) is dict:
+            final_return_result = {}
+            for key, value_ in return_result.items():
                 if hasattr(value_, 'value'):
-                    return_result[key] = value_.value()
+                    final_return_result[key] = value_.value()
                 else:
-                    return_result[key] = value_.value()
-        elif type(self._value) is list:
-            return_result = []
-            for item in self._value:
+                    final_return_result[key] = value_.value()
```

Comment:
Should this be 

```
final_return_result[key] = value_
```

Seems like both branches are the same

Reply:
Good catch! The other branch should never happen. I replaced it with an exception.